test/mpi/include/mpithreadtest.h

```diff
@@ -28,7 +28,12 @@
 #if !defined(THREAD_PACKAGE_NAME)
 #error "thread package (THREAD_PACKAGE_NAME) not defined"
 
-#elif THREAD_PACKAGE_NAME == THREAD_PACKAGE_WIN
+#elif THREAD_PACKAGE_NAME == THREAD_PACKAGE_NONE
+/* Empty. No threaded tests should run. */
+
+#else /* valid THREAD_PACKAGE_NAME */
+
+#if THREAD_PACKAGE_NAME == THREAD_PACKAGE_WIN
```

Comment:
This can be merged as an `#elif` for consistency.

Reply:
The `#else` clause used to extend to the function prototypes below. But I agree that straight `elif` branches are much easier to read. Modified the code accordingly.